### PR TITLE
Updating plan names.

### PR DESCRIPTION
**Pull request by lanceleonard**
This is a highly targeted update designed to only update subscription plan names.  (OK, I also updated the headings to reflect current style).

The article will receive a development edit in a future work item.

### CODE REVIEW

File: content/rc/security/database-security/tls-ssl.md [hunk 1]
```diff
@@ -7,17 +7,15 @@ categories: ["RC"]
 aliases: /rc/administration/security/securing-redis-cloud-connections
 ---
 
-Transport Layer Security (TLS), commonly called "SSL", ensures the privacy of data sent between applications and their Redis databases. TLS is available in Redis Cloud Pro and Redis Cloud Ultimate.
-
-## Using TLS with Redis Cloud
+Transport Layer Security (TLS), commonly called "SSL", ensures the privacy of data sent between applications and their Redis databases. TLS is available in Redis Cloud Flexible and Annual subscriptions.
+\
```
Comment (banker) [inline content/rc/security/database-security/tls-ssl.md:11]: @lanceleonard Is this "\" intentional?

**Review by banker — APPROVED** [review]
See my one comment

File: content/rc/security/database-security/tls-ssl.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/security/database-security/tls-ssl.md]: Not by me.  :-)

(I hadn't done a copy edit on the content.  Was trying to only update the plan names.  The more recent commits include a quick copy edit and updated images for the certificate authorization buttons.)

File: content/rc/security/database-security/tls-ssl.md [hunk 2]
```diff
@@ -7,81 +7,81 @@ categories: ["RC"]
...
 {{</note>}}
 
-## Connecting over TLS
+## Connect over TLS
 
 To connect to a Redis Cloud database over TLS, you will need:
 
 * A Redis client that supports TLS
-* The Redis Cloud CA certificate
+* The Redis Cloud client authentication (CA) certificate
```
Comment (banker) [inline content/rc/security/database-security/tls-ssl.md:68]: Here, CA looks like an abbreviation for "client authentication", but it's not. I'd remove the parentheses.

File: content/rc/security/database-security/tls-ssl.md [hunk 3]
```diff
@@ -7,81 +7,81 @@ categories: ["RC"]
...
-console by going to **Settings** and clicking on **Pro CA**.
+If you don't have the Redis Cloud CA certificate, you can download it from the admin
+console by going to **Settings** and selecting the **Flexible Plans CA** button.
 
-![Pro CA](/images/rc/pro-ca.png "Pro CA")
+![Flexible Plans CA button](/images/rc/rc-settings-ca-flexible.png "Flexible Plans CA")
 
-In addition, if you're requiring client authentication, then you'll need a public and private client key. See
-[Enabling TLS](#enabling-tls) for where to get these files.
+If you're requiring client authentication, you'll also need a public and private client keys. See
```
Comment (banker) [inline content/rc/security/database-security/tls-ssl.md:77]:
```suggestion
If you're requiring client authentication, you'll also need public and private client keys. See
```

File: content/rc/security/database-security/tls-ssl.md [hunk 4]
```diff
@@ -7,81 +7,81 @@ categories: ["RC"]
...
+[Enabling TLS](#enabling-tls) for detai.
 
-### Connecting with the Redis CLI
+### Connect with the Redis CLI
 
-Here's a quick example of how to connect to your TLS-enabled Redis Cloud database using the Redis CLI.
+Here's how to use the Redis command-line interface (CLI) tool to a TLS-enabled Redis Cloud database:
```
Comment (banker) [inline content/rc/security/database-security/tls-ssl.md:82]:
```suggestion
Here's how to use the Redis CLI to connect to a TLS-enabled Redis Cloud database:
```

File: content/rc/security/database-security/tls-ssl.md (hunk 2 above)
Comment (lanceleonard) [reply content/rc/security/database-security/tls-ssl.md:68]: Yes; that's intentional.

File: content/rc/security/database-security/tls-ssl.md (hunk 3 above)
Comment (lanceleonard) [reply content/rc/security/database-security/tls-ssl.md:77]: Fixed.

File: content/rc/security/database-security/tls-ssl.md (hunk 4 above)
Comment (lanceleonard) [reply content/rc/security/database-security/tls-ssl.md:82]: Fixed.

File: content/rc/security/database-security/tls-ssl.md (hunk 2 above)
Comment (banker) [inline content/rc/security/database-security/tls-ssl.md:68]:
```suggestion
* The Redis Cloud CA certificate
```

Comment (lanceleonard) [reply content/rc/security/database-security/tls-ssl.md]: Fixed.

**Pull request merged**